Papa 7 Moorings/Gliders & AUV Metadata Review

cdobs opened the pull request:
-Recovery information added for FLMA/FLMB/HYPM D6
-Deployment information added for FLMA/FLMB/HYPM D7
-Deployment information added for GL537 D2 and PG575 D2

spetillo commented on deployment/GP05MOAS-GL537_Deploy.csv:
```diff
@@ -1,9 +1,9 @@
+SKQ201920S,,,,GP05MOAS-GL537-00-ENG000000,2,1,2019-09-30T01:40:00,,CGVGP-05MOAS-00000,CGVEH-GLDROO-00537,CGCON-GLDROO-00537,50.00878,-144.37213,,1000,1000,
+SKQ201920S,,,,GP05MOAS-GL537-01-FLORDM000,2,1,2019-09-30T01:40:00,,CGVGP-05MOAS-00000,CGVEH-GLDROO-00537,CGINS-FLORDM-03582,50.00878,-144.37213,,1000,1000,
+SKQ201920S,,,,GP05MOAS-GL537-02-DOSTAM000,2,1,2019-09-30T01:40:00,,CGVGP-05MOAS-00000,CGVEH-GLDROO-00537,CGINS-DOSTAM-00401,50.00878,-144.37213,,1000,1000,
+SKQ201920S,,,,GP05MOAS-GL537-04-CTDGVM000,2,1,2019-09-30T01:40:00,,CGVGP-05MOAS-00000,CGVEH-GLDROO-00537,CGINS-CTDGVM-09237,50.00878,-144.37213,,1000,1000,
```
S/N's in UIDs also look good, checked against FCP & tail configuration record.

spetillo commented on deployment/GP05MOAS-PG575_Deploy.csv:
```diff
@@ -1,15 +1,15 @@
+SKQ201920S,,,,GP05MOAS-PG575-03-FLORTM000,2,1,2019-09-30T01:47:00,,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00575,CGINS-FLORTM-03815,50.0114167,-144.37255,,200,200,
+SKQ201920S,,,,GP05MOAS-PG575-04-FLORTO000,2,1,2019-09-30T01:47:00,,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00575,CGINS-FLORTO-01332,50.0114167,-144.37255,,200,200,
+SKQ201920S,,,,GP05MOAS-PG575-05-NUTNRM000,2,1,2019-09-30T01:47:00,,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00575,CGINS-NUTNRM-00711,50.0114167,-144.37255,,200,200,
+SKQ201920S,,,,GP05MOAS-PG575-06-PARADM000,2,1,2019-09-30T01:47:00,,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00575,CGINS-PARADM-50200,50.0114167,-144.37255,,200,200,
```
S/N's in UIDs also look good, checked against FCP & tail configuration record.

cdobs commented:
I added two files to this PR: CP05MOAS-A6263_Deploy.csv and CP05MOAS-A6264_Deploy.csv and have request re-review for these two files only. 

I completed a review of all deployment metadata for our AUVs and added in any deployments that were missing. These files should have been placed in a separate branch but I added them here by mistake. After talking with @spetillo we decided to just roll with it and review them here because it's just 2 files and the work to correct my mistake would take longer than just reviewing them here. I'd like to get them through ASAP so I can bring AUV ingestion up to date. 

AUV deployment times can be cross-referenced with the QLR for that cruise, and also in our AUV tracking sheet, which is where the SNs can be cross-referenced.

spetillo commented on deployment/CP05MOAS-A6263_Deploy.csv:
```diff
@@ -1,57 +1,78 @@
+WJ20180423,,WJ20180423,,CP05MOAS-A6263-03-CTDAVN000,6,1,2018-04-24T13:05:00,2018-04-24T22:42:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-CTDAVN-00424,40.0167,-71.0967,,,,along-shelf survey
+WJ20180423,,WJ20180423,,CP05MOAS-A6263-04-NUTNRN000,6,1,2018-04-24T13:05:00,2018-04-24T22:42:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-NUTNRN-01109,40.0167,-71.0967,,,,along-shelf survey
+WJ20180423,,WJ20180423,,CP05MOAS-A6263-05-ADCPAN000,6,1,2018-04-24T13:05:00,2018-04-24T22:42:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-ADCPAN-22076,40.0167,-71.0967,,,,along-shelf survey
+WJ20180423,,WJ20180423,,CP05MOAS-A6263-06-PARADN000,6,1,2018-04-24T13:05:00,2018-04-24T22:42:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-PARADN-50267,40.0167,-71.0967,,,,along-shelf survey
```
@cdobs These cruise numbers (WJ20180423) do not match the ones in your AUV Tracking 2.0 spreadsheet. Based on the date in this cruise number, I think it is correct here and incorrect on your spreadsheet (which says "P10a (WJ20180705)"). Can you verify?

cdobs replied:
Yup, it should be WJ20180423, I fixed the spreadsheet!

spetillo commented on deployment/CP05MOAS-A6264_Deploy.csv:
```diff
@@ -1,71 +1,99 @@
+WJ20180705,,WJ20180705,,CP05MOAS-A6264-03-CTDAVN000,6,1,2018-04-24T06:40:00,2018-04-24T16:40:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-CTDAVN-00425,39.9184,-70.8985,,,,across-shelf survey
+WJ20180705,,WJ20180705,,CP05MOAS-A6264-04-NUTNRN000,6,1,2018-04-24T06:40:00,2018-04-24T16:40:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-NUTNRN-01121,39.9184,-70.8985,,,,across-shelf survey
+WJ20180705,,WJ20180705,,CP05MOAS-A6264-05-ADCPAN000,6,1,2018-04-24T06:40:00,2018-04-24T16:40:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-ADCPAN-22077,39.9184,-70.8985,,,,across-shelf survey
+WJ20180705,,WJ20180705,,CP05MOAS-A6264-06-PARADN000,6,1,2018-04-24T06:40:00,2018-04-24T16:40:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-PARADN-50266,39.9184,-70.8985,,,,across-shelf survey
```
@cdobs These cruise numbers (WJ20180705) are suspect... They are after the deployment date of this AUV (2018-04-24).

cdobs replied:
Updated to WJ20180423 which is consistent with the tracking sheet now

spetillo commented on deployment/CP05MOAS-A6263_Deploy.csv:
```diff
@@ -1,57 +1,78 @@
+AR-39,,AR-39,,CP05MOAS-A6263-03-CTDAVN000,11,1,2019-10-02T12:10:00,2019-10-02T22:36:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-CTDAVN-00424,40.205267,-71.0940167,,,,abbreviated mission due to time constraints (AL-1 to AL-2)
+AR-39,,AR-39,,CP05MOAS-A6263-04-NUTNRN000,11,1,2019-10-02T12:10:00,2019-10-02T22:36:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-NUTNRN-00366,40.205267,-71.0940167,,,,abbreviated mission due to time constraints (AL-1 to AL-2)
+AR-39,,AR-39,,CP05MOAS-A6263-05-ADCPAN000,11,1,2019-10-02T12:10:00,2019-10-02T22:36:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-ADCPAN-22076,40.205267,-71.0940167,,,,abbreviated mission due to time constraints (AL-1 to AL-2)
+AR-39,,AR-39,,CP05MOAS-A6263-06-PARADN000,11,1,2019-10-02T12:10:00,2019-10-02T22:36:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-PARADN-50170,40.205267,-71.0940167,,,,abbreviated mission due to time constraints (AL-1 to AL-2)
```
Entire file reviewed and updated by @spetillo and @cdobs. Corrections made as we found issues.

spetillo commented on deployment/CP05MOAS-A6264_Deploy.csv:
```diff
@@ -1,71 +1,99 @@
+AR-39,,AR-39,,CP05MOAS-A6264-03-CTDAVN000,14,1,2019-09-30T12:03:00,2019-09-30T21:59:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-CTDAVN-00425,39.92068,-70.764267,,,,abbreviated mission due to time constraints (AC-4 to AC-1)
+AR-39,,AR-39,,CP05MOAS-A6264-04-NUTNRN000,14,1,2019-09-30T12:03:00,2019-09-30T21:59:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-NUTNRN-00367,39.92068,-70.764267,,,,abbreviated mission due to time constraints (AC-4 to AC-1)
+AR-39,,AR-39,,CP05MOAS-A6264-05-ADCPAN000,14,1,2019-09-30T12:03:00,2019-09-30T21:59:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-ADCPAN-22077,39.92068,-70.764267,,,,abbreviated mission due to time constraints (AC-4 to AC-1)
+AR-39,,AR-39,,CP05MOAS-A6264-06-PARADN000,14,1,2019-09-30T12:03:00,2019-09-30T21:59:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-PARADN-50180,39.92068,-70.764267,,,,abbreviated mission due to time constraints (AC-4 to AC-1)
```
Entire file reviewed and updated by @spetillo and @cdobs. Corrections made as we found issues.
